modules/tinymce/src/themes/silver/test/ts/browser/editor/backstage/BackstageCheckContextTest.ts

```diff
@@ -91,6 +91,23 @@ describe('browser.tinymce.themes.silver.editor.backstage.BackstageSinkTest', ()
           assert.deepEqual(lazyBackstages().popup.shared.providers.checkUiComponentContext('mode:!random'), { contextType: 'mode', shouldDisable: false });
           assert.deepEqual(lazyBackstages().popup.shared.providers.checkUiComponentContext('mode:!testmode'), { contextType: 'mode', shouldDisable: true });
         });
+
+        it('TINY-13143: Handle multiple context entries', async () => {
+          const editor = hook.editor();
+          editor.ui.registry.addContext('context_1', (value) => value === 'test');
+          editor.ui.registry.addContext( 'context_2', (value) => value === 'test');
+
+          assert.deepEqual(lazyBackstages().popup.shared.providers.checkUiComponentContext('context_1:test,context_2:test'), { contextType: 'context_1,context_2', shouldDisable: false });
+
+          assert.deepEqual(lazyBackstages().popup.shared.providers.checkUiComponentContext('context_1:test,context_2:test2'), { contextType: 'context_1,context_2', shouldDisable: true });
+
+          // Ignore empty entry
+          assert.deepEqual(lazyBackstages().popup.shared.providers.checkUiComponentContext('context_1:test,,context_2:test2,'), { contextType: 'context_1,context_2', shouldDisable: true });
+
+          // context_3 is not registered so it is reasonable to expect shouldDisabled:true
+          // However, because of the current logic, any unfound context is assumed as mode:design
```

Comment:
I personally don't mind the fallback to `mode:design`. Although, I do think it needs to be updated slightly if we are supporting multiple contexts. Could we refactor the code so that contexts that don't exist are first filtered out. If there are no contexts to iterate over, then can fallback to `mode:design'.

Reply:
Done!